Workflow: build and manage a resilient service using Python (#5343)
awsdocs/aws-doc-sdk-examples: workflows/resilient_service/README.md

> # Build and Manage a Resilient Service using an AWS SDK

Reviewer: ```suggestion
# Build and manage a resilient service using an AWS SDK
```

## Overview

This example shows how to use AWS SDKs to create a load-balanced

> web service that returns recommendations of books, movies, and songs. It demonstrates
> how the service responds to failures, and shows ways to restructure the service to be
> more resilient when failures occur.

Reviewer: ```suggestion
web service that returns book, movie, and song recommendations. It shows
how the service responds to failures, and how to restructure the service for
more resilience when failures occur.
```

Several components are used to demonstrate the resilience of the example web service:

* [Amazon EC2 Auto Scaling](https://docs.aws.amazon.com/autoscaling/ec2/userguide/what-is-amazon-ec2-auto-scaling.html) 
  is used to create 
  [Amazon Elastic Compute Cloud (Amazon EC2)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html) 
  instances based on a launch template and to ensure that the number of instances is kept 
  in a specified range.
* [Elastic Load Balancing](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html) 
  handles HTTP requests, monitors the health of instances in the Auto Scaling group, and 
  dispatches requests to healthy instances. 
* A Python web server runs on each EC2 instance to handle HTTP requests. It responds
  with recommendations and health checks.
* An [Amazon DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Introduction.html) 
  table simulates a recommendation service that the web server depends on to get recommendations.
* A set of [AWS Systems Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html) 

>   parameters that control how the web server responds to requests and health checks to 
>   simulate failures and demonstrate resiliency. 

Reviewer: ```suggestion
  parameters control web server response to requests and health checks to 
  simulate failures and demonstrate resiliency. 
```
